app/components/hero-block.js

```diff
@@ -106,13 +108,21 @@ const useStylesFromThemeFunction = createUseStyles(theme => ({
       height: '64vw',
     },
   }),
-  subjectWrapper: {
+
+  subjectWrapper: props => ({
     position: 'absolute',
     top: '50%',
     transform: 'translateY(-50%)',
-    textAlign: 'center',
-    width: '100%',
-  },
+    textAlign: props.alignContent,
+    width: props.alignContent === 'center' ? '100%' : 'calc(100% - 4rem)', // 减去左右内边距
```

Comment:
comments are useful, but can this be in english? I think it says subtract left and right padding which explains why 4rem which is a good comment.